src/NodaTime.TzdbCompiler/NodaTime.TzdbCompiler.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
TzdbCompiler is an application, not a library. This change at least should be reverted (or at least changed).

Reply:
Absolutely. I thought you'd get compilation error when trying to compile an exe targeting `netstandard2.0`, but apparently you don't!